### PR TITLE
In Firefox, only the latest version of flash is supported

### PR DESCRIPTION
**Pull request by hghazzi**
When the current version of flash is not the latest in firefox, then it
should fall back to html.

***need to add unit tests***

JW7-2839

### CODE REVIEW

**Comment by robwalch** [comment]
@hghazzi looks like these changes interfere with this unit test on FF:
test/unit/playlist-filtering-test.js in commercial

**Comment by robwalch** [comment]
Try /jwplayer-commercial/test/qunit/?module=playlist.filterPlaylist on your local in FF.

**Comment by hghazzi** [comment]
test this please

**Comment by hghazzi** [comment]
test this please

**Comment by hghazzi** [comment]
test this please

File: src/js/utils/browser.js [hunk 1]
```diff
@@ -14,6 +15,17 @@ define([
         };
     }
 
+    function _parseVersion(version) {
+        var parts = version.split(/[\s,. ]+/);
```
Comment (robwalch) [inline src/js/utils/browser.js]: `N,i.c e`!

**Pull request merged**